README.md

```diff
@@ -394,18 +418,21 @@ user experience:
    has to possible choose between the Firefox browser and
    a malicious app that has registered itself for the
    domain name. 
+   JJH: i'm guessing only firefox has this behaviour? Is it resolved in fenix? ( https://www.ctrl.blog/entry/review-firefox-fenix-android.html )
```

Comment:
With Fenix you mean the Firefox Nightly browser right? I just tested it and the problem does not occur in the Firefox Nightly browser.

Reply:
According to https://arewefenixyet.com the normal Firefox on the App Store is Fenix everywhere except US. (That only happening during this morning.)

Reply:
I just updated my Firefox browser and now it looks the same as the Firefox Nightly browser and this problem is gone. So we can probably remove this section from the document.